BEST_PRACTICES.md

```diff
@@ -151,6 +154,10 @@ file).
 Schemas
 -------
 
+Taps should describe the format of an entity's expected data using JSON schema.
+
+To understand our reasoning for this choice, read [Data Types and Schemas](SCHEMAS.md)
```

Comment:
"our" is weird here - there's no reference to "you" or "us" anywhere else in the doc.  Recommend changing to "To understand the rationale for using schemas, ..."